tests/scripts/check-generated-files.sh

```diff
@@ -132,6 +132,7 @@ check tests/scripts/generate_bignum_tests.py $(tests/scripts/generate_bignum_tes
 check tests/scripts/generate_ecp_tests.py $(tests/scripts/generate_ecp_tests.py --list)
 check tests/scripts/generate_psa_tests.py $(tests/scripts/generate_psa_tests.py --list)
 check tests/scripts/generate_test_keys.py tests/src/test_keys.h
+check tests/scripts/generate_test_cert_macros.py tests/src/test_certs.h
```

Comment:
I don't think this is the right place: we're in the list of things common to Mbed TLS and TF-PSA-Crypto, but certs (X.509) are part of Mbed TLS, not PSA Crypto, so this line should probably go in "Additional checks for Mbed TLS only" below.